data/avro-schema/src/main/resources/avro/net/corda/data/persistence/DeleteEntities.avsc

```diff
@@ -0,0 +1,16 @@
+  {
+    "type": "record",
+    "name": "DeleteEntities",
+    "doc": "Delete a specific set of entities.  See also {@link EntityRequest#request}",
+    "namespace": "net.corda.data.persistence",
+    "fields": [
+      {
+        "name": "entityArray",
```

Comment:
```suggestion
        "name": "entities",
```

We use `entities` in the merge operation for example (and `ids` for Delete By Id). I don't mind which way, but let's keep it consistent.

Reply:
Done